### DISCUSSION TITLE
Support for Firefox Running on Android

### DISCUSSION THREAD

**Discussion by ThePCMedic** [aclap-dev/video-downloadhelper#127] [Help and Q&A]
### Your question

Greetings!
I have been a user of Video Download Helper for quite some time. It is an awesome add-on!
Fairly recently, Mozilla released a version of Firefox that, once again, works on the Android O/S. However, this version of Firefox doesn't support all the same add-ons that Firefox running under Windows and Linux do.

I am curious to know if VDH is currently able to be added to Firefox on Android, or if work is underway to make whatever adaptations are needed so that it can be added to Firefox running on Android.

Many thanks for your time and consideration

**Comment by jcfrog** [comment]
Hi,
The VDH  extension as is cannot work on Android for many technical reasons. 
Maybe a good surprise in the following months ?

**Reply by 11111one11111** [reply]
Any update on an extension for firefox on android?

**Reply by jcfrog** [reply]
Not for the moment, sorry.
